templates/web/base/admin/_report-assignment.html

```diff
@@ -1,4 +1,4 @@
-[%  IF c.user.has_permission_to('report_inspect', problem.bodies_str_ids ) %]
+[%  IF c.user.has_body_permission_to('report_inspect') OR c.user.has_body_permission_to('assign_report_to_user') %]
```

Comment:
The template that includes this e.g. item_small uses `c.user.has_permission_to('report_inspect', problem.bodies_str_ids)` so I would keep that if you have a problem to check against, rather than switch to has_body_permission_to.